### PR TITLE
Add my solution to HTML CSS exercise

### PR DESCRIPTION
**Pull request by sennator2**
```
This is a PR template, If you are adding a solution link to the curriculum leave this as is. If you are not, then delete it and write the message you wish.
Thank you,
The Odin Project team.

```

### CODE REVIEW

**Comment by KevinMulhern** [comment]
Perfect, thanks @sennator2 LGTM 👍

File: web_development_101/project_html_css.md [hunk 1]
```diff
@@ -94,6 +94,7 @@ There are a couple of ways to go about doing this, but the simplest is this:
 *Submit a link below to the github repo with your files in it by using a pull request.  See the section on [Contributing](http://github.com/TheOdinProject/curriculum/blob/master/contributing.md) for how.  Please include your partner's github handle somewhere in the description if you had one and they would like attribution.*
 
 Add your solution below this line!
+*[Danilo's solution (Easy)](https://sennator2.github.io/google-homepage/)  | [View in browser (Easy)]( https://sennator2.github.io/google-homepage/)
```
Comment (arun1595) [inline web_development_101/project_html_css.md]: Looks like you've not linked to your github repo.

Comment (sennator2) [inline web_development_101/project_html_css.md]: OK now I' m not sure what you mean?

Comment (arun1595) [inline web_development_101/project_html_css.md]: @sennator2 Both the links you've added are the same. Usually, people add a link to the repository and if possible, a link to the live preview.

**Pull request merged**